completions/fish/rmw.fish

```diff
@@ -0,0 +1,30 @@
+# This file contains the fish completion for rmw
+#
+# rmw is a trashcan/recycle bin utility for the command line
+#
+# More information about this tool on https://github.com/theimpossibleastronaut/rmw
+#
+# This file is intended to be released with rmw with various packaging managers
+
+complete rmw -s h -l help -d 'display help' -f
+complete rmw -s c -l config -d 'use an alternate configuration' -r
+complete rmw -s l -l list -d 'list waste directories' -f
+complete rmw -s g -l purge -d 'purge expired files'
+complete rmw -s o -l orphaned -d 'check for orphaned files (maintenance)'  -f
+complete rmw -s f -l force -d 'allow purging of expired files' -f
+complete rmw -l empty -d 'completely empty (purge) all waste directories'  -f
+complete rmw -l top-level-bypass -d 'bypass protection of top-level files'
+
+complete rmw -s v -l verbose -d 'Explain what is done' -f
+complete rmw -l version -d 'Display rmw version' -f
+complete rmw -l warranty -d 'display warranty' -f
+
+complete rmw -s z -l restore -d 'restore FILE(s)'
+complete rmw -s s -l select -d 'interactively select files from list to restore' -f
+complete rmw -s u -l undo-last -d 'undo last deletion' -f
+complete rmw -s m -l most-recent-list -d 'list files most recently deleted with rmw' -f
```

Comment:
```suggestion
complete rmw -s m -l most-recent-list -d 'list most recently rmw\'ed files' -f
```

They're not deleted. 

The term `rmw'ed` may not be the best  but I hope it's clear enough. What do you think?

Is there a reason you didn't use the strings from the help output exactly as they appear?

Reply:
I should have reused your exact terms, but yes.

I have to confess now I'm caught, that the terms were unclear to me. That's right, they are not deleted or removed, rmw puts them away.

It's the same behavior as the trash on desktop whatever the OS.

I enjoyed the fact you tool allows to use the  "regular trash" on Linux.

Your tool name is rmw, and you insist on the move part with the "ReMove to wasted"

You also mention your tool is "a trashcan/recycle bin utility"

Wording is a pain, I'm not good at it.

We shouldn't used "wasted files", as it would be connoted. So maybe, simply "trashed".

But on contrary, the `--list` talk about "waste directories"

So, choice is uneasy. And it cannot be mine only.

Anyway, I will sync the wording with the one in --help and if you want to reword, it can be done in another PR. It would include the readme, code and completion

Reply:
Wording can be difficult. :) For this PR, I agree we won't do any rewording.

Thanks for the feedback. I'm glad you like the program.

Reply:
I reused your wordings